python-client/giskard/core/model_validation.py

```diff
@@ -250,3 +253,13 @@ def validate_classification_prediction(classification_labels: Union[np.ndarray,
         )
     if prediction.shape[1] != len(classification_labels):
         raise ValueError("Prediction output label shape and classification_labels shape do not match")
+
+
+def validate_order_classifcation_labels(model, dataset):
+    from giskard.testing.tests.performance import test_accuracy
+    result = test_accuracy(model=model, dataset=dataset).execute()
+    if result.metric <= 0.4:
```

Comment:
for binary classification we can even do a more clever check by trying to flip the order ourselves and see if it improves the metric

Reply:
@rabah-khalek @andreybavt I think there just an easy, definitive check that we can do: balanced accuracy. For a random predictor, it should always be $1 / N_\text{classes}$ (right @rabah-khalek?).
Thus if we find balanced accuracy < $1 / N_\text{classes}$ it means that the model is worse than a random predictor, i.e. is actively making bad predictions which means that labels must be inversed.